### DISCUSSION TITLE
More options for shipyards

### DISCUSSION THREAD

**Discussion by TIKIRobo** [tsunamayo/Starship-EVO#4123] [Suggestions]
These are three basic suggestions for shipyards, feel free to add more.
**The ability to set shipyards to a more precise mode** 
where the ship docking would move in such a way that its center point/center of mass lines up with the center of the shipyard's bounds (and then moves strait down if its set to be a landing pad)  
**Random Ship Offset**
this would be like the precise mode except you could randomly make it land a slightly different spot, this way you can have control where it lands but still have that "human touch" 
**Random Ship Rotation**
as an extension to the random ship offset it would be interesting to see ships able to park a little sideways on the landing pad, that way (if you set it to the correct angles) the autodock wouldn't have to spin the ship around a bunch to get it to face the right way

**Comment by KINGrimson** [comment]
over all better ai for when it is trying to dock, its clunky and moves around way to much to make any good hangar work, you need twice the size od your ship if you want it to dock properly
